server/src/main/java/io/crate/exceptions/RoleAlreadyExistsException.java

```diff
@@ -23,9 +23,25 @@
 
 import java.util.Locale;
 
+import org.jetbrains.annotations.Nullable;
+
+import io.crate.role.JwtProperties;
+
 public class RoleAlreadyExistsException extends RuntimeException implements ConflictException, UnscopedException {
 
-    public RoleAlreadyExistsException(String roleName) {
-        super(String.format(Locale.ENGLISH, "Role '%s' already exists", roleName));
+    private RoleAlreadyExistsException(String message) {
+        super(message);
+    }
+
+    public static RoleAlreadyExistsException of(String roleName, @Nullable JwtProperties jwtProperties) {
```

Comment:
minor: I would pass a boolean flag, to denote the kind of the exception and not a jwtProperties arg.